test/user.test.js

```diff
@@ -122,4 +130,31 @@ describe('Users', () => {
         .catch(err => done(err));
     });
   });
+  describe('GET /users', () => {
+    beforeEach(async () => {
+      try {
+        await request(app)
+          .post('/users')
+          .send(inputUserExpected);
+      } catch (error) {
+        logger.error(error.errors);
+      }
+    });
+
+    test('Paginated user list should be returned', async done => {
+      const getAllUsersSpy = jest.spyOn(UserService, 'getAllUsers');
+      getAllUsersSpy.mockImplementation(getUsersListMock);
+      await request(app)
+        .get('/users?offset=0&limit=5')
+        .set('Authorization', `Bearer ${testToken}`)
+        .set('Accept', 'application/json')
+        .expect(200)
+        .then(res => {
+          expect(res.body.data.users.count).toBe(countMock);
+          expect(res.body.data.users.rows).toEqual(rowsMock);
+          done();
+        })
+        .catch(err => done(err));
+    });
+  });
```

Comment:
You could add more cases, for example when the token is not sent or when you add the schema that validates the parameters of the query you could add a test when letters are sent instead of numbers.